plugin/core/views.py

```diff
@@ -781,28 +783,49 @@ def run(self, view_id: int, command: str, args: Optional[Dict[str, Any]] = None)
 
 
 COLOR_BOX_HTML = """
-<style>html {{padding: 0; background-color: transparent}}</style>
+<style>
+    html {{
+        padding: 0;
+        background-color: transparent;
+    }}
+    a {{
+        display: inline-block;
+        height: 0.8rem;
```

Comment:
What do you think about 0.9rem for the height and width?

Reply:
Personally I'd prefer to keep it at 0.8rem. It's probably only 1 pixel difference, but with the font size and screen resolution I use there seems only to be 1 pixel between adjacent color boxes on consecutive lines when using 0.9rem.

But I would be okay with increasing them to 0.9rem if bigger color boxes are preferred by the majority.

Reply:
I've tried both and I don't really have strong preference either way.